skins/oasis/modules/templates/PageHeader_Index.php

```diff
@@ -7,12 +7,20 @@
 	}
 }
 
+if ( $showCuratedContentToolBtn ) {
+	$curatedContentToolButton = '<a id="CuratedContentTool" class="wikia-button">' . wfMsg( 'wikiacuratedcontent-edit-mobile-main-page' ) . '</a>';
```

Comment:
On the other note - Creating snippets like that seems wrong - have you considered creating new function in controller and separate template for this?